packages/flutter/lib/src/scheduler/scheduler.dart

```diff
@@ -18,7 +18,7 @@ double timeDilation = 1.0;
 /// scheduler's epoch. Use timeStamp to determine how far to advance animation
 /// timelines so that all the animations in the system are synchronized to a
 /// common time base.
-typedef void SchedulerCallback(Duration timeStamp);
+typedef void FrameCallback(Duration timeStamp);
```

Comment:
I will leave it in this CL, but will follow up.
Since onBeginFrame is public, we might as well make its type public too, and then we should probably have only one type.
However, it requires two move the Engine and Flutter forward in sync, so I prefer doing that change in separate PRs.